### PR TITLE
feat(feedback): add spam detection to project settings

### LINKED ISSUE #66731: Add proj options to settings

### PR DESCRIPTION
**Pull request by michellewzhang**
Create a User Feedback section under the Processing header, to add a spam detection toggle in project settings. We already have a User Feedback section under SDK Setup, but I didn't think that spam fit in with SDK setup. The result is that we now have two User Feedback sections in the settings (open to ideas for consolidating or renaming):
SCR-20240311-nuse

The new page looks like this:
SCR-20240311-numj

Closes https://github.com/getsentry/sentry/issues/66731

### CODE REVIEW

File: static/app/data/forms/userFeedbackProcessing.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import type {JsonFormObject} from 'sentry/components/forms/types';
+
+export const route = '/settings/:orgId/projects/:projectId/user-feedback-processing/';
```
Comment (ryan953) [inline static/app/data/forms/userFeedbackProcessing.tsx:3]: I can't find an import for this one

**Review by ryan953 — APPROVED** [review]

**Review by billyvg — APPROVED** [review]
That menu is starting to get unwieldly, but I don't have a good solution

File: static/app/data/forms/userFeedbackProcessing.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/data/forms/userFeedbackProcessing.tsx:3]: oops, copied from a previous file, but in those i see this:
`// Export route to make these forms searchable by label/help`

so maybe it's necessary

**Comment by michellewzhang** [comment]
> That menu is starting to get unwieldly, but I don't have a good solution

too many settings!

**Pull request merged**